sql/moz-fx-data-shared-prod/search/desktop_mobile_clients_daily_bizdev_report/view.sql

```diff
@@ -0,0 +1,34 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shared-prod.search.desktop_mobile_clients_daily_bizdev_report`
```

Comment:
Is this the appropriate name for this table now? It seems like we generalized a bit, for e.g. `firefox_products_search_clients_engines_sources_daily`. Should we include `sources` in this table as well?